### PR TITLE
scala: add start interpreter with classpath

### PR DESCRIPTION
**Pull request by serragnoli**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/scala.md [hunk 1]
```diff
@@ -7,6 +7,10 @@
 
 `scala`
 
+- Start the interpreter with a fat jar in the classpath:
```
Comment (bl-ue) [inline pages/common/scala.md:10]: What does 'fat' mean here? :smile:

Comment (serragnoli) [reply pages/common/scala.md:10]: I meant a jar with all its dependencies (other jars) inside it. But to be honest, I think "dependency" is a better word for that. I'll push an update. 😅

**Review by bl-ue — APPROVED** [review]
LGTM.  Thanks @serragnoli!

File: pages/common/scala.md [hunk 2]
```diff
@@ -7,6 +7,10 @@
 
 `scala`
 
+- Start the interpreter with a dependency in the classpath:
+
+`scala -cp {{filename.jar}} {{command}}`
```
Comment (navarroaxel) [inline pages/common/scala.md:12]: We prefer long argument when is possible:
```suggestion
`scala -classpath {{filename.jar}} {{command}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @serragnoli, I made a comment.

**Review by bl-ue — APPROVED** [review]

**Review by sbrl — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**